modules/Link.js

```diff
@@ -22,6 +22,10 @@ function isEmptyObject(object) {
   return true
 }
 
+function resolveToLocation(to, router) {
```

Comment:
Nevermind, we're still using `createClass`, so that's not even possible. I meant something like this:

```js
get to() {
  const { to } = this.props
  return typeof to === 'function' ? to(this.context.router.location) : to
}

handleClick(event) {
  //...
  router.push(this.to)
}
```